student.rb

```diff
@@ -1,26 +1,20 @@
 require './person'
+require './classroom'
+
 class Student < Person
-  def initialize(classroom, age, name, parent_permission)
-    @classroom = classroom
+  def initialize(classroom_label, age, name, parent_permission)
+    @classroom = Classroom.new(classroom_label)
```

Comment:
- Great Job by trying to add new improvements in your code however I this is better to bring back the code as it was before is more logical instead of having a variable and trying to rename it in another way you can just use     `@classroom = classroom` is easier.  and as it is the same as you used in all other places :+1: